tests/simplecheck.py

```diff
@@ -2,12 +2,8 @@
 
 import humanify
 import datetime
-import sys
 
 test_date = datetime.datetime(2018, 9, 4, 0, 40, 20)
 
-if not humanify.datetime(test_date) == 'Tuesday, 4th of September 2018 00:40:20':
+if not humanify.datetime(test_date) == "Tuesday, 4th of September 2018 00:40:20":
     raise Exception("Unit test gave incorrect results")
-    sys.exit(1)
```

Comment:
Does raising an exception cause the exit code 1? 

Please refer to this: https://tldp.org/LDP/abs/html/exitcodes.html

Reply:
Don't think it's needed as tests typically error out, returning other then code `0`